index.js

```diff
@@ -0,0 +1,120 @@
+const inquirer = require("inquirer");
+const figlet = require('figlet');
+const mysql = require("mysql2");
+const cTable = require("console.table");
+
+const connect = require("./db/db");
+var db;
+
+const ALL_CLASSES = {
+  Employee: require("./lib/Employee"),
+  Role: require("./lib/Role"),
+  Department: require("./lib/Department")
+};
+
+const MAIN_MENU = {
+  'View All Employees': { className: 'Employee', method: 'getInfo', callBack: infoCallBack },
+  'View Salaries By Department': { className: 'Department', method: 'getSalariesByDepartment', callBack: infoCallBack },
+  'View Employees By Department': { className: 'Employee', method: 'getEmployeesByDeparment', callBack: infoCallBack },
+  'View Employees By Manager': { className: 'Employee', method: 'getEmployeesByManager', callBack: infoCallBack },
+  'Add Employee': { className: 'Employee', method: 'add', callBack: putCallBack },
+  'Update Employee Role': { className: 'Employee', method: 'updateRole' , callBack: putCallBack },
+  'View All Roles': { className: 'Role', method: 'getInfo', callBack: infoCallBack },
+  'Add Role': { className: 'Role', method: 'add', callBack: putCallBack },
+  'View All Departments': { className: 'Department', method: 'getInfo', callBack: infoCallBack },
+  'Add Department': { className: 'Department', method: 'add', callBack: putCallBack },
+  'Delete Employee': { className: 'Employee', method: 'delete', callBack: putCallBack },
+  'Delete Role': { className: 'Role', method: 'delete', callBack: putCallBack },
+  'Delete Department': { className: 'Department', method: 'delete', callBack: putCallBack },
+  'Quit': {}
+};
+
+const QUESTIONS = [
+  {
+    type: "list",
+    name: "request",
+    message: "What would you like to do?",
+    choices: Object.keys(MAIN_MENU)
+  },
+  {
+    type: "confirm",
+    name: "confirm_quit",
+    message: "Are you sure you want to quit?",
```

Comment:
Great holistic thinking here! Really like this addition to affirm that the user wants to complete the operation